web/public/styles.css

```
@@ -0,0 +1,107 @@
+.badge.red { color: var(--red); border-color: rgba(248,81,73,0.4); }
+.badge.yellow { color: var(--yellow); border-color: rgba(210,153,34,0.4); }
+.badge.blue { color: var(--blue); border-color: rgba(88,166,255,0.4); }
+.score { display: inline-block; min-width: 36px; text-align: center; font-weight: 600; padding: 2px 6px; border-radius: 4px; }
+.score.high { background: rgba(63,185,80,0.15); color: var(--green); }
+.score.mid { background: rgba(210,153,34,0.15); color: var(--yellow); }
+.score.low { background: rgba(248,81,73,0.15); color: var(--red); }
+.actions { display: flex; gap: 4px; }
+.actions button { padding: 4px 8px; font-size: 12px; }
+.url { font-family: ui-monospace, SFMono-Regular, Menlo, monospace; font-size: 11px; word-break: break-all; max-width: 360px; display: inline-block; }
+.filters-grid { display: grid; grid-template-columns: repeat(auto-fit, minmax(220px, 1fr)); gap: 12px; }
+.filters-grid .card pre { margin: 6px 0 0; font-size: 11px; color: var(--muted); white-space: pre-wrap; word-break: break-word; }
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)